rust/examples/delay_producer.rs

```diff
@@ -23,8 +23,8 @@ use rocketmq::Producer;
 
 #[tokio::main]
 async fn main() {
-    // recommend to specify which topic(s) you would like to send message to
-    // producer will prefetch topic route when start and failed fast if topic not exist
+    // recommend specifying which topic(s) you would like to send message to
```

Comment:
It's recommended to specify the topics that applications will publish messages to because the producer will prefetch topic routes for them on start and fail fast in case they do not exist